sos/plugins/openstack_glance.py

```diff
@@ -76,7 +72,10 @@ def setup(self):
 
     def running_in_container(self):
         for runtime in ["docker", "podman"]:
-            container_status = self.get_command_output(runtime + " ps")
+            container_status = self.get_cmd_output(
```

Comment:
to call `self.get_cmd_output_now` instead? :)

Reply:
Typo, yep - thanks for catching that.